python/ray/experimental/serve/kv_store_service.py

```diff
@@ -172,6 +172,8 @@ def as_dict(self):
 class RoutingTable:
     def __init__(self, kv_connector):
         self.routing_table = kv_connector("routing_table")
+        self.no_route_service_table = kv_connector("no_route")
```

Comment:
do we need a new table here? is it possible that we can coalesce it into routing table?

Reply:
Yeah! Let me try coalescing it